web/wp-content/themes/lfevents/src/assets/scss/modules/_event-footer.scss

```diff
@@ -27,13 +27,12 @@
 .event-footer-newsletter {
   margin-bottom: 3rem;
 
-  .legal-consent-container {
```

Comment:
Can you not delete this? From time-to-time the LF team adds legal text on these newsletter forms and it shows visible on the site unless we have this.

Reply:
Ah ok. Can you add a note to it? Just looks like random unused css otherwise.